Sources/Reports.php

```diff
@@ -288,6 +288,8 @@ function BoardReport()
  */
 function BoardPermissionsReport()
 {
+	global $context, $txt;
```

Comment:
There's already a global statement in this function, which includes $txt. No need to add another one just for one variable.

Reply:
I'll add a third one just to make you happy.